cmd/describe/cmd.go

```diff
@@ -21,12 +21,14 @@ import (
 
 	"github.com/openshift/moactl/cmd/describe/addon"
 	"github.com/openshift/moactl/cmd/describe/cluster"
+	v "github.com/openshift/moactl/cmd/validations"
 )
 
 var Cmd = &cobra.Command{
-	Use:   "describe RESOURCE [flags]",
-	Short: "Show details of a specific resource",
-	Long:  "Show details of a specific resource",
+	Use:              "describe RESOURCE [flags]",
+	Short:            "Show details of a specific resource",
+	Long:             "Show details of a specific resource",
+	PersistentPreRun: v.Validations,
```

Comment:
I'm not sure the `describe` command should do this prerun validation, as it just reads from OCM based on the configured AWS credentials (it doesn't actually interact with the AWS resources)

Reply:
Addressed and resolved